service/storage_service.cc

```diff
@@ -587,6 +589,17 @@ future<> storage_service::sync_raft_topology_nodes(mutable_token_metadata_ptr tm
     }
 
     co_await when_all_succeed(sys_ks_futures.begin(), sys_ks_futures.end()).discard_result();
+
+    co_return nodes_to_notify;
+}
+
+future<> storage_service::notify_nodes_after_sync(nodes_to_notify_after_sync&& nodes_to_notify) {
+    for (auto ip : nodes_to_notify.left) {
```

Comment:
Is the order important here? Could/should this be `coroutine::parallel_for_each`?

Reply:
Not sure, but I'd rather not risk introducing flakiness. There aren't that many lifecycle subscribers in the whole system and the events aren't emitted very frequently, so there isn't much performance to save.